src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  & > :not(:nth-child(n+5)) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
move the star logic form previous task to make it flexible with different stars modifier